src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -652,6 +658,13 @@ public void setBannedRoutes(String s) {
             bannedRoutes = RouteMatcher.emptyMatcher();
     }
 
+    public void setWhiteListedRoutes(String s) {
+        if (s != null && !s.equals(""))
+            whiteListedRoutes = RouteMatcher.parse(s);
+        else
```

Comment:
Please use brackets even around single-line blocks, unless the body is on the same line as its introductory keyword. E.g.:
```
if (x=1) {
    callSomething();
} else {
    continue;
}
```

or

```
if (x=1) callSomething();
else continue;
```
If this is not part of our style guide we should update it.

Of course this is debatable but was hashed out on the mailing list a few years back.